### PR TITLE
Configuration console command

### PR DESCRIPTION
**Pull request by misantron**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Configuration console command to display config settings

## Issue ticket number and link

https://github.com/bumble-tech/bumble-doc-gen/issues/48

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [ ] I have added/updated tests.

### CODE REVIEW

**Comment by misantron** [comment]
@shcherbanich should configuration command support all configuration keys or just collections?

**Comment by shcherbanich** [comment]
Ideally, this command should support the output of all keys

File: src/Core/Configuration/Configuration.php [hunk 1]
```diff
@@ -195,7 +195,7 @@ public function getCacheDir(): ?string
         } catch (ObjectNotFoundException) {
         }
 
-        $cacheDir = $this->parameterBag->validateAndGetStringValue('cache_dir');
+        $cacheDir = $this->parameterBag->validateAndGetStringValue(ConfigurationKey::CACHE_DIR);
         $parentDir = dirname($cacheDir);
         if (!is_dir($parentDir)) {
             throw new InvalidConfigurationParameterException(
```
Comment (seanmcn) [inline src/Core/Configuration/Configuration.php:201]: Can you change `cache_dir` in the message here to `ConfigurationKey::CACHE_DIR`?

**Review by seanmcn — CHANGES_REQUESTED** [review]
Overall it looks good thank you! 

I've requested some minor changes around logging messages. 

I also think it would be nice if the command could output all configuration, as a user will be unlikely to iniitally know the config keys available. 

Finally, there are merge conflicts on the branch (due to the generated documentation), so please  merge `1.x.x` into the branch again & regenerate the documentation.

Thanks :)

**Review by seanmcn — APPROVED** [review]

**Comment by seanmcn** [comment]
@misantron thank you for your contribution!

**Pull request merged**